testsuite/MDAnalysisTests/analysis/test_encore.py

```diff
@@ -429,6 +429,11 @@ def ens2(self, ens2_template):
             ens2_template.filename,
             ens2_template.trajectory.timeseries(order='fac'),
             format=mda.coordinates.memory.MemoryReader)
+
+    @pytest.fixture()
+    def empty(self):
```

Comment:
nitpick but would you be able to give this a slightly more descriptive name? Perhaps `empty_cluster`

Reply:
I'll update it, no problem